.github/workflows/ci.yml

```diff
@@ -1,5 +1,5 @@
-on: push
-name: New workflow
+name: CI
+on: [push, pull_requests]
 jobs:
```

Comment:
I don't see `pull_requests` event in docs, only `pull_request`

Reply:
Thanks!